amaxa/amaxa.py

```diff
@@ -110,6 +111,8 @@ def close(self):
         for f in self.store.values():
             f.close()
 
+def _format_traceback(exception):
```

Comment:
I usually place private helper functions at the bottom of the files after the public ones. What do you think about it?